build.rs

```diff
@@ -48,6 +48,9 @@ fn main() {
     if let Some(rev) = git_revision_hash() {
         println!("cargo:rustc-env=RIPGREP_BUILD_GIT_HASH={}", rev);
     }
+
+    // embed windows application manifest file
+    embed_resource::compile("rg-manifest.rc");
```

Comment:
What does this do on non-Windows targets? Even if nothing, it might be nice to gate this on `#[cfg(windows)]` to make it a bit clearer to those reading the code.